spinalcordtoolbox/download.py

```diff
@@ -50,6 +50,9 @@ def download_data(urls):
                 _, content = cgi.parse_header(response.headers['Content-Disposition'])
                 filename = content["filename"]
 
+            # protect against directory traversal
+            filename = os.path.basename(filename)
```

Comment:
Does this need to worry about symlinks?

Reply:
so far we don't have symlinks to download, so it should be fine?

Reply:
This was to protect against [directory traversal](https://owasp.org/www-community/attacks/Path_Traversal).

My thought about symlinks is a related but obscure attack:

1. The mirror -- or someone who manages to impersonate it -- knows that you have a symlink in your download folder, say `keys -> ~/.ssh/authorized_keys`, 
2. They return "Content-Disposition: attachment; filename=keys" with the contents "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKYKsUxEseqswXVJ3LbmMq6eXkluXNdU6EszFUBx8LUN attacker@evil"
3. They can now ssh into your machine.